core/controllers/galleries.py

```diff
@@ -113,6 +113,10 @@ def get(self):
         explorations_list = (
             summary_services.get_displayable_exp_summary_dicts_matching_ids(
                 exp_ids))
+        # TODO(mgowano): Remove '' in query to allow for collection searches
+        collections_list = (
+            collection_services.get_displayable_collections_matching_query(
```

Comment:
(I don't know the context, but just jumping in to say that #1697 _might_ be related. /cc @BenHenning )